common-libs/common/src/shared/EcdsaCryptographyTools.ts

```diff
@@ -0,0 +1,51 @@
+import { Secp256k1Key, Secp256k1Signature } from '@affinidi/tiny-lds-ecdsa-secp256k1-2019'
+import { IPlatformCryptographyTools } from './interfaces'
+
+export const ecdsaCryptographyTools: IPlatformCryptographyTools = {
+  signSuites: {
+    bbs: () => {
+      throw new Error('Not implemented')
+    },
+    rsa: () => {
+      throw new Error('Not implemented')
+    },
```

Comment:
It's a bit confusing that these are here. 
Why not have `signSuite`, `generateKey`, `verifySuiteFactory` here and don't mention bbs/rsa?

What I mean is, imho, we should have the opposite:
```
{
    rsa: { signSuite, generateKey, verifySuiteFactory },
    ecdsa: { signSuite, generateKey, verifySuiteFactory },
    bbs: { signSuite, generateKey, verifySuiteFactory, deriveSegmentProof, validateSegmentProof },
}
```

Therefore we won't have to mention bbs/rsa in this ecdsa helper file.

Reply:
1. We mention bbs/rsa in ecdsa helper file because `Affinity.ts` uses these, and the alternative would be to make them optional so that `Affinity.ts` will have to check that the object is not undefined and the function is not undefined prior to calling it, and throw the more cryptic error otherwise. With the current implementation, there are no optional fields, code in `Affinity.ts` does not require any additional checks, and exceptions contain informative traces.

2. Note that `keyGenerators` only has `rsa` and `bbs` fields, while for `verifySuiteFactories` keys are entirely different. It is not just "verify a ecdsa signature", it is specifically "verify signature for credential with `proof.type === 'EcdsaSecp256k1Signature2019'`.

3. And this also makes code simpler in `Affinity.ts`: when it needs to verify a signature, it does not have to concern itself with "what proof type corresponds to what cryptographic function" (and what if there are several proof types relying on the same cryptographic function?), it can just get a verifier for proof type.